src/bin/pg_autoctl/cli_do_tmux.c

```diff
@@ -635,7 +647,7 @@ tmux_pg_autoctl_create_postgres(PQExpBuffer script,
 	tmux_add_send_keys_command(script, "export PGPORT=%d", pgport);
 
 	sformat(monitor, sizeof(monitor),
-			"$(%s show uri --pgdata %s/monitor --monitor)",
+			"$(%s show uri --pgdata %s/monitor --formation monitor)",
```

Comment:
I think if we change the args like this we would need a minor release instead of a patch. Since it could be a breaking change for some automation.

Reply:
Also, maybe we should disallow using `monitor` as an actual formation name.

Reply:
Agreed about the minor release, it's a breaking change, although a very minor one. In another PR coming soon we reserve the monitor name for a formation where we register the monitor node itself...